.github/workflows/lock.yml

```diff
@@ -11,7 +11,9 @@ on:
     #     default: "true"
 
   schedule:
-    - cron: 0 6 * * *
+    # on the hour, every 24 hours
+    # https://crontab.guru/#0_5_*_*_*
+    - cron: 0 5 * * *
```

Comment:
I'm not sure I follow, the distance between the stale and lock workflows was supposed to be longer to make sure timeouts could run out, but you're trying to move the runtime out of the US working hours?

Reply:
So I don't think its a collision with the stale workflows that are causing the timeout, I think its a collision with the issue-tracker workflows ([recent runs hourly](https://github.com/conda/issue-tracker/blob/main/.github/workflows/recent.yml), [monthly runs every 3 hours](https://github.com/conda/issue-tracker/blob/main/.github/workflows/month.yml), [yearly runs at 6](https://github.com/conda/issue-tracker/blob/main/.github/workflows/year.yml), [all-time runs at 6 every 3 days](https://github.com/conda/issue-tracker/blob/main/.github/workflows/alltime.yml)).

Reply:
We could instead try to change the issue-tracker schedules, changing the lock schedule was just less work 🤷🏻‍♂️

Reply:
Gotcha, I know it's more work, but could we just reduce the runtime of the issue-tracker workflow of recent and monthly to daily, too?